Alexander_Borisov_L2.py

```diff
@@ -0,0 +1,366 @@
+# HOME WORK 3.1
+
+print('\n----------------\nHOME WORK 3.1\n----------------\n')
+
+n1 = n2 = n3 = n4 = 0
+max_value = 0
+max_number = ' '
+
+print('Enter 4 numbers (N1, N2, N3, N4)...')
+
+# set 1-st number as max
+n1 = float(input('N1 = '))
+max_value = n1
+max_number = 'N1'
+
+# replace if any of next numbers is bigger
+n2 = float(input('N2 = '))
+if n2 > max_value:
+    max_value = n2
+    max_number = 'N2'
+
+n3 = float(input('N3 = '))
+if n3 > max_value:
+    max_value = n3
+    max_number = 'N3'    
+
+n4 = float(input('N4 = '))
+if n4 > max_value:
+    max_value = n4
+    max_number = 'N4'    
+
+print('Max number is', max_number, '=', max_value) 
```

Comment:
Can be simplified just like that:
n1 = 
n2 = 
n3 = 
n4 = 

And use the if/elif/else notation